src/schema/v2/me/myCollection.ts

```diff
@@ -86,9 +86,7 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
       return null
     }
 
-    const paginationArgs = options.sortByLastAuctionResultDate
-      ? { size: 100 }
-      : convertConnectionArgsToGravityArgs(options)
+    const paginationArgs = convertConnectionArgsToGravityArgs(options)
```

Comment:
Would the following work as well? Not sure whether we need the `size: 100` for some reason.

```suggestion
    const paginationArgs = options.sortByLastAuctionResultDate
    ? { ...convertConnectionArgsToGravityArgs(options), size: 100 }
    : convertConnectionArgsToGravityArgs(options)
```

Reply:
This does not fix the bug, unfortunately.
With this approach there is also a high chance of passing conflicting params that are not in sync at all. For eg, requesting for the last 3 while also requesting for a size of 100.
Note, We automatically derive `limit` from these options and that would inform the `size`.